README.rst

```diff
@@ -5,18 +5,18 @@
    :target: https://travis-ci.org/Pylons/pyramid_mailer
 
 .. image:: https://readthedocs.org/projects/pyramid_mailer/badge/?version=latest
-   :target: http://docs.pylonsproject.org/projects/pyramid_mailer/en/latest/
+   :target: https://docs.pylonsproject.org/projects/pyramid_mailer/en/latest/
    :alt: Documentation Status
 
 pyramid_mailer is a package for sending email from your Pyramid application.
-It is compatible with Python 2.7, 3.3, 3.4, and 3.5, as well as PyPy.
+It is compatible with Python 2.7, 3.4, 3.5, and 3.6, as well as PyPy.
 
 This package includes:
 
 1. Wrapping the low-level Python ``email`` library with an easy-to-use
    API, which includes attachments and mulipart content.
 
-2. Send emails immediately or to add to a maildir queue.
+2. Send emails immediately or add to a ``maildir`` queue.
```

Comment:
Prefer matching the mode of the others (`Sending` like `Wrapping`, `Managing`).

Reply:
Do you mean remove the double ticks or change to single ticks? In reST single ticks render as italic/emphasis, and double ticks render as inline code or arguments, but in markdown single ticks render as inline code.

Reply:
I mean the verb at the start of the sentence should be in the same mode / tense as the ones in the other two items:

```
2. Sending emails immediately....
```

Reply:
Done in 0976410.